File: 1. Two Sum/solution.go
func twoSum(nums []int, target int) []int {
    myMap := make(map[int]int)
    for i := 0;i < len(nums);i++ {
        if _, ok := myMap[target - nums[i]];ok {
            return []int{myMap[target - nums[i]], i}
        }
        myMap[nums[i]] = i
    }
    return nil
}

File: 1. Two Sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numIndex = {}  # store (num, index) pair in dict
        for i, num in enumerate(nums):
            if target - num in numIndex:
                return (numIndex[target - num], i)
            else:
                numIndex[num] = i
